hooks/pre_gen_project.py

```diff
@@ -28,4 +32,13 @@
         {{ cookiecutter.update({ '_tailwind_addon': 'yes' }) }}
     {% endif %}
 {% endif %}
+
+-----------
+OpenAPI
+cookiecutter.use_openapi
+-----------
+Only project with web only:
```

Comment:
How about (`only` is duplicated 🙈 ):

```suggestion
Only for projects using the web variant:
```